_data/sponsorship.yml

```
@@ -0,0 +1,151 @@
+benefits:
+  - type: Sponsor virtual room
+    diamond: 4
+    platinum: 2
+    gold: 2
+    silver: 1
+    startup: 1
+  - type: Sponsor chat room (Zulip)
+    diamond: 4
+    platinum: 3
+    gold: 2
+    silver: 1
+    startup: no
+  - type: Logo, Write-up, link on conference website
+    diamond: yes
+    platinum: yes
+    gold: yes
+    silver: yes
+    startup: yes
+  - type: Complimentary conference session passes
+    diamond: 15
+    platinum: 10
+    gold: 6
+    silver: 4
+    startup: 3
+  - type: A hallway track named after sponsor
+    diamond: yes
+    platinum: no
+    gold: no
+    silver: no
+    startup: no
+  - type: Mention in the opening session
+    diamond: yes
+    platinum: yes
+    gold: no
+    silver: no
+    startup: no
+  - type: Mention in the closing session
+    diamond: yes
+    platinum: yes
+    gold: yes
+    silver: yes
+    startup: yes
+  - type: Logo on final released talk videos
+    diamond: yes
+    platinum: yes
+    gold: no
+    silver: no
+    startup: no
+  - type: Mention on the website
+    diamond: yes
+    platinum: yes
+    gold: yes
+    silver: yes
+    startup: yes
+  - type: Mention on the banner (in between talks)
+    diamond: Fixed
+    platinum: Fixed
+    gold: Rotating
+    silver: Rotating
+    startup: no
+  - type: Advertisement on program guide
+    diamond: Double Page
+    platinum: Single Page
+    gold: Half Page
+    silver: no
+    startup: no
+  - type: Direct marketing Video ads (up to 20 seconds)
+    diamond: 8
+    platinum: 6
+    gold: 2
+    silver: 1 (10 sec)
+    startup: no
+  - type: Virtual Swag: PDF
```

In YAML, you have to encapsulate values containing `:` with quotes.

```suggestion
  - type: "Virtual Swag: PDF"
```